server/notification-providers/slack.js

```diff
@@ -27,10 +27,14 @@ class Slack extends NotificationProvider {
 
     async send(notification, msg, monitorJSON = null, heartbeatJSON = null) {
         let okMsg = "Sent Successfully.";
+        const finalMsg = notification.slackchannelmention
+            ? `${msg} <!channel>`
+            : msg;
```

Comment:
I would prefer this if the current usage is intentional.

```suggestion
        if (notification.slackchannelmention) {
             msg += " <!channel>";
        }
```